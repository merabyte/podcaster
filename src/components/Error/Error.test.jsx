import { render, screen } from '@testing-library/react'
import { createMemoryRouter, RouterProvider } from 'react-router-dom'

import Error from './'

let router

describe('Error', () => {
  beforeEach(() => {
    router = createMemoryRouter(
      [
        {
          path: '/',
          element: <Error />,
        },
      ],
      { initialEntries: ['/'], initialIndex: 0 },
    )
  })

  afterEach(() => {
    router = null
  })

  it('should match snapshot', () => {
    const { container } = render(<RouterProvider router={router} />)
    expect(container).toMatchSnapshot()
  })
})
