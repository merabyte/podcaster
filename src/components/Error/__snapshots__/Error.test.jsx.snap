// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Error > should match snapshot 1`] = `
<div>
  <div>
    <h1>
      Oops!
    </h1>
    <p>
      Sorry, an unexpected error has occurred.
    </p>
    <p>
      <i />
    </p>
  </div>
</div>
`;
